lib/domain/app_state/room/create_room_success.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:fluffychat/app_state/success.dart';
+
+enum CreateRoomSuccessEventType {
+  createRoomSuccess,
+  setAvatarSuccess,
+}
+
+class CreateRoomSuccess extends Success {
+  final CreateRoomSuccessEventType event;
+  final bool isDone;

```

Comment:
why I need it? I am not a fan of it. How about to create a dedicated `success` for it.

Reply:
Made a new CreateRoomSuccessEventType

Reply:
why not anothe instance of `Success`?